dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserController.java

```diff
@@ -217,7 +255,12 @@ private UserQueryParams toUserQueryParams(GetUserObjectListParams params) {
     res.setInvitationStatus(params.getInvitationStatus());
     res.setUserOrgUnits(params.isUserOrgUnits());
     res.setIncludeOrgUnitChildren(params.isIncludeChildren());
-    res.setOrgUnitBoundary(params.getOrgUnitBoundary());
+    String ou = params.getOu();
```

Comment:
FYI: moving this code here is in a way a bugfix so that the `orgUnitBoundary` parameter can be used and is not always overridden by the effect of `addOrganisationUnit`.